prov/efa/src/efa_base_ep.h

```diff
@@ -27,6 +27,19 @@ struct efa_qp {
 	uint32_t qkey;
 };
 
+/* raw address format. (section 1.4) */
```

Comment:
since you moved the code, you need to update the comment as now reader doesn't related `section 1.4` to the protocol doc.

Reply:
removed the outdated comment.